Lib/abc.py

```diff
@@ -121,6 +121,49 @@ def _abc_caches_clear(cls):
             """Clear the caches (for debugging or testing)."""
             _reset_caches(cls)
 
+def update_abstractmethods(cls):
+    """Repopulate the abstract methods of an abstract class, or a subclass of
+    an abstract class.
```

Comment:
I would like the summary to fit on one line.
```suggestion
    """Recalculate the set of abstract methods of an abstract class.
```